gp-includes/route.php

```diff
@@ -311,10 +311,10 @@ public function set_notices_and_errors() {
 	/**
 	 * Loads a template.
 	 *
-	 * @param string $template template name to load
-	 * @param array $args Associative array with arguements, which will be exported in the template PHP file
+	 * @param string      $template template name to load
+	 * @param array       $args Associative array with arguements, which will be exported in the template PHP file
 	 * @param bool|string $honor_api If this is true or 'api' and the route is processing an API request
-	 * 		the template name will be suffixed with .api. The actual file loaded will be template.api.php
+	 *      the template name will be suffixed with .api. The actual file loaded will be template.api.php
```

Comment:
This should be aligned with where the description starts at "If"

Reply:
Done, also aligned comments and added fullstops.